// Copyright (c) 2020 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_grpc_protos;

message ProcessInfo {
  uint32 pid = 1;
  string name = 2;
  double cpu_usage = 3;
  string full_path = 4;
  string command_line = 5;
  bool is_64_bit = 6;
  string build_id = 7;
}
